kong/plugins/ip-restriction/handler.lua

```diff
@@ -52,31 +79,45 @@ local function match_bin(list, binary_remote_addr)
 end
 
 
-function IpRestrictionHandler:access(conf)
+local function handler(conf)
   local binary_remote_addr = ngx_var.binary_remote_addr
   if not binary_remote_addr then
-    return kong.response.error(403, "Cannot identify the client IP address, unix domain sockets are not supported.")
+    local status = 403
+    local message = "Cannot identify the client IP address, unix domain sockets are not supported."
+
+    do_exit(status, message)
   end
 
   local deny = conf.deny
   local allow = conf.allow
   local status = conf.status or 403
-  local message = conf.message or "Your IP address is not allowed"
+  local default_message = string.format("IP address not allowed: %s", binary_remote_addr)
```

Comment:
We can not output `binary_remote_addr` as `%s`, perhaps we should use the original message `Your IP address is not allowed`

Reply:
Sorry, we had fixed this in our code and I forgot to copy it over.  I changed it to the string address variable.  We would really like to keep this feature.  It's been very useful when trouble shooting with users and identifying holes in the allow lists.

Reply:
@chronolaw is the fix acceptable?